assets/app/views/about.html

```diff
@@ -0,0 +1,65 @@
+<div>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="about">
+          <div class="row">
+            <div class="col-md-2 about-icon gutter-top hidden-sm hidden-xs">
+              <img src="images/openshift-logo.svg" />
+            </div>
+            <div class="col-md-9">
+              <div class="page-header page-header-bleed-right">
+                <h1>OpenShift by Red Hat&reg;</h1>
+              </div>
+
+              <h2>About</h2>
+              <p><a href="https://openshift.com">OpenShift</a> is Red Hat's Platform-as-a-Service (PaaS) that allows developers to quickly develop, host, and scale applications in a cloud environment.</p>
+              
+              <h2 id="version">Version</h2>
+              <dl class="dl-horizontal left">
+                <dt>OpenShift Master:</dt>
+                <dd>{{version.master.openshift || 'unknown'}}</dd>
+                <dt>Kubernetes Master:</dt>
+                <dd>{{version.master.kubernetes || 'unknown'}}</dd>
+                <dt>OpenShift API:</dt>
+                <dd>{{version.api.openshift}}</dd>
+                <dt>Kubernetes API:</dt>
+                <dd>{{version.api.kubernetes}}</dd>
+              </dl>
+
+              <h2 id="cli">Command Line Tools</h2>
+              <p>
+                With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal.
+                <span ng-if="cliDownloadURLPresent">
+                  You can download the <code>oc</code> client tool using the links below. For more information about downloading and installing it, please refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation.
+                </span>
+                <span ng-if="!cliDownloadURLPresent">
+                  Refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation for instructions about downloading and installing the <code>oc</code> client tool.
+                </span>
+                <div ng-if="cliDownloadURLPresent">
+                  <label class="cli-download-label">Download <code>oc</code>:</label>
+                  <div ng-repeat="(key, value) in cliDownloadURL">
+                    <a href="{{value}}" class="cli-download-link">
+                      {{key}}
+                      <i class="fa fa-external-link"></i>
+                    </a>
+                  </div>
+                </div>
+              </p>
+              <p>
+                After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
+                <pre class="code prettyprint ng-binding" ng-if="sessionToken">
+oc login {{loginBaseURL}} --token={{sessionToken}}
+</pre>
+                <pre class="code prettyprint ng-binding" ng-if="!sessionToken">
+oc login {{loginBaseURL}} --token={{sessionToken}}
```

Comment:
Hrm, this is a `<pre`... should we click-to-reveal the entire box? I was thinking about something like below (where "show" is the link), but we will need to use a `div` instead of `pre` and apply the styles.

```
oc login https://localhost:8443 --token=<show>
```